data/policies/parent_call.cas

```diff
@@ -33,9 +33,30 @@ resource xyz inherits abc {
 	}
 }
 
-// TODOs once we add the ability to cast to
-// non-inherited things
-//
-//domain asd {
-//	asd<abc>.read(this);
-//}
+domain asd {
```

Comment:
The name of this file implies that it's only the "parent" version of casting.  I don't really mind having this here, but we should rename the file to call_casting or something like that.  I'd also be fine if you prefer putting this stuff in a new test file.